orchestra/static/orchestra/common/css/_common.scss

```diff
@@ -74,3 +74,124 @@ $main-font-stack: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helveti
      transform:rotate(360deg);
```

Comment:
sanity-checking we should be updating timer.css (maybe) and orchestra.css (probably)

Reply:
I think all are necessary to render the various pages. I updated _common.scss to make sure the css was loaded on every page, and gulp did the rest.